ui/src/components/TabbedQueriesBuilder/index.js

```diff
@@ -80,29 +80,29 @@ class TabbedQueriesBuilder extends Component {
           onChange={this.props.onChangeLabel.bind(null, index)} />
       </div>
 
-      {this.renderQueryBuilder(index)}
+      {this.renderQueryBuilder(index, circlePicker)}
 
     </span>
   );
 
-  renderQuery = index => (
+  renderQuery = ( index, circlePicker = true) => (
```

Comment:
Not sure what this circle picker stuff is but it feels like it shouldn't be a thing for this PR.